CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Technical - Installs salsprep required by mongo to sanitize passwords.
```

Comment:
1. Is it a feature?
2. Isn't there any end user behavior change that could be described here (instead of a technical description)?